internal/disk/disk.go

```diff
@@ -14,6 +14,11 @@ import (
 	"github.com/osbuild/osbuild-composer/internal/osbuild2"
 )
 
+const (
+	// Default sector size in bytes
+	DefaultSectorSize = 512
```

Comment:
Could this ever change per-distro? I.e., will we see a future where the default is 4096 for some but not all distros?

Reply:
Maybe, it will be moved to the `PartitionTable` in a subsequent commit, which will make it more flexible.

Reply:
Ah, ignore this comment.